c-sharp-razor/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Base image for the runtime (using Alpine)
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS base
+WORKDIR /app
+EXPOSE 5000
```

Comment:
Why don't you declare it in the bottom of the file after build? Instead of redeclaring it `FROM base AS final`

Reply:
Addressed comment